### PR TITLE
update nxos show lag to allow for second line

### PR DESCRIPTION
**Pull request by mgdransfield**
## Description

update nxos show lag to parse second line of interfaces

## Motivation and Context

fixes nxos show lag to parse second lne

## Impact (If any)

none

## Checklist:

- [x ] I have updated the changelog.
- [ ] I have updated the documentation (If applicable).
- [x ] I have added tests to cover my changes (If applicable).
- [ x] All new and existing tests passed.
- [ x] All new code passed compilation.

### CODE REVIEW

File: src/genie/libs/parser/iosxe/show_license.py [hunk 1]
```diff
@@ -72,7 +72,7 @@ def cli(self, output=None):
         #         License Priority: None
         p6 = re.compile(r"\s+(?P<license_priority>\S+)")
         #         License State: Active, Not in Use, EULA not accepted
```
Comment (tahigash) [inline src/genie/libs/parser/iosxe/show_license.py:74]: need example output as example for regex change

File: src/genie/libs/parser/nxos/show_lag.py [hunk 2]
```diff
@@ -328,9 +328,11 @@ def cli(self, output=None):
             r'(?P<bundle_id>[\d]+)[\xa0 ]+(?P<name>[\w\-]+)\((?P<flags>[\w]+)\)?[\xa0 '
             r']+(?P<type>\w+)[\xa0 ]+(?P<protocol>[\w\-]+)?[\xa0 ]+(?P<ports>[\w\-/() '
             r'\xa0]+ *)?$')
+        p2 = re.compile(
```
Comment (tahigash) [inline src/genie/libs/parser/nxos/show_lag.py:331]: Please have example output as comment for the change of p2 regex.

File: src/genie/libs/parser/iosxe/show_license.py [hunk 3]
```diff
@@ -28,15 +28,15 @@ class ShowLicenseSchema(MetaParser):
         'licenses': {
             int: {
                 'feature': str,
-                'period_left': str,
+                Optional('period_left'): str,
```
Comment (tahigash) [inline src/genie/libs/parser/iosxe/show_license.py:31]: could you please add unittest for the output which we need to make these as `Optional`?

**Review by tahigash — CHANGES_REQUESTED** [review]
reviewed and left comments

File: src/genie/libs/parser/nxos/show_lag.py (hunk 2 above)
Comment (mgdransfield) [reply src/genie/libs/parser/nxos/show_lag.py:332]: added example output

File: src/genie/libs/parser/iosxe/show_license.py (hunk 3 above)
Comment (mgdransfield) [reply src/genie/libs/parser/iosxe/show_license.py:31]: added unittest with items needed optional

**Review by mgdransfield — COMMENTED** [review]
made requested changes

File: src/genie/libs/parser/iosxe/show_license.py (hunk 1 above)
Comment (tahigash) [reply src/genie/libs/parser/iosxe/show_license.py:75]: confirmed the change

**Review by tahigash — APPROVED** [review]
thanks for addressing comments. schema looks good now.

**Review by choudnik — APPROVED** [review]

**Pull request merged**